load-testing/config/load_test_manifest_reader.go

```diff
@@ -11,15 +11,23 @@ import (
 type ProvisionedActorConfig struct {
 	// KeyName is the **name** of the key in the keyring to be used by the given actor.
 	KeyName string `yaml:"key_name"`
+	// Address is the address of the actor, which is used to identify already staked
+	// actors in the network in persistent chains.
+	Address string `yaml:"address"`
 	// ExposedUrl is the URL where the actor is expected to be reachable.
 	ExposedUrl string `yaml:"exposed_url"`
 }
 
 // LoadTestManifestYAML is the struct which the load test manifest is deserialized into.
 // It contains the list of suppliers and gateways that the load test expects to be pre-provisioned.
 type LoadTestManifestYAML struct {
-	Suppliers []ProvisionedActorConfig `yaml:"suppliers"`
-	Gateways  []ProvisionedActorConfig `yaml:"gateways"`
+	// PersistentChain is a flag that indicates whether the test is expected to be
+	// run on localnet or long living chains (i.e. TestNet/DevNet).
+	PersistentChain bool                     `yaml:"persistent_chain"`
```

Comment:
I'm open to having a discussion about this because it'll involve updating the whole PR, but I'd rather we switch the whole thing from `PersistentChain` to `EphemeralChain`.

- I think the default is that chains are persistent.
- When I read this, I was confused by what a persistent chain is.
- I'd argue that "ephemeral chain" is a lot more clear than "persistent chain" and self explanatory of the alternatives.

Reply:
I prefer `ephemeral chain` too, I'll update the code.